compose/ui/ui/src/desktopMain/kotlin/androidx/compose/ui/ComposeFeatureFlags.desktop.kt

```diff
@@ -85,7 +85,24 @@ internal object ComposeFeatureFlags {
      * - On macOS, render and event dispatching order differs. It means that interop view might
      *   catch the mouse event even if visually it renders below Compose content
      */
-    val useInteropBlending: Boolean by lazy {
+    val useInteropBlending = FeatureFlag {
```

Comment:
Let's continue to use delegated properties here to avoid "value" suffix.
You just need to add `operator fun getValue` in `FeatureFlag` class. See [Delegated properties](https://kotlinlang.org/docs/delegated-properties.html)

Reply:
🤔 Right.. I just think that it's better to avoid changing usage pattern across the project.

whyt about this API shape?
```kt
internal object ComposeFeatureFlags {

    @VisibleForTesting
    internal fun withOverride(
        layerType: LayerType = this.layerType,
        useSwingGraphicsInComposePanel: Boolean = this.useSwingGraphicsInComposePanel,
        useInteropBlending: Boolean = this.useInteropBlending,
        block: () -> Unit,
    ) { ... } 
```

- No changes in existing usage
- Explicit restriction for testing via `VisibleForTesting`
- Single scope in case of multiple overrides

Reply:
True. Anyway, doesn't seem as blocker